ceph.spec.in

```diff
@@ -81,24 +81,20 @@ BuildRequires:	python
 BuildRequires:	python-argparse
 BuildRequires:	python-nose
 BuildRequires:	python-requests
-%if ( 0%{?rhel} > 0 && 0%{?rhel} < 7 ) || ( 0%{?centos} > 0 && 0%{?centos} < 7 )
-BuildRequires:	python-sphinx10
-%endif
```

Comment:
These macros for family_redhat or family_suse seem more specific, is their a reason for this change.